zulipterminal/model.py

```diff
@@ -115,8 +115,12 @@ def __init__(self, controller: Any) -> None:
         (self.stream_dict, self.muted_streams,
          self.pinned_streams, self.unpinned_streams) = stream_data
 
+        # NOTE: The expected response has been upgraded from
+        # [stream_name, topic] to [stream_name, topic, date_muted] in
+        # 3.0, Feature level 1.
```

Comment:
It'd be useful to standardize our notes in the code for this; we could add that to the development section - this would make it easier to check for changes at different server versions and feature levels :)